src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE = 0;
+    private static final int OPERATION_TYPE_RESULT = 2;
+    private static final int OPERATION_TYPE_BUY = 1;
+    private static final int OPERATION_TYPE_SUPPLY = 0;
```

Comment:
you already have this constant value

Reply:
OPERATION_TYPE is used to get a type supply or buy
OPARTION_TYPE_SUPPLY is used when we know type for sure

should it really be changed?